app/views/casa_cases/_form.html.erb

```diff
@@ -12,6 +12,16 @@
         <% end %>
       </div>
 
+      <div class="field form-group">
+        <% if current_user.casa_admin? || current_user.supervisor?  %>
+          <%= form.label :court_date, "Next Court Date" %>
+          <br>
+          <span class="datetime-year-month"> <%= form.date_select :court_date, {order: [:day, :month, :year], start_year: Date.current.year, end_year: 1989, prompt: {day: 'Day', month: 'Month', year: 'Year'} }, class: "select2 date-input"%></span>
```

Comment:
This is giving us a year range from 1989 - 2020, but we actually need years to go at least a couple into the future.

Reply:
We can probably make the start year 2020, unless they are going to backdate any records...? @pollygee ?

Reply:
@littleforest  Court dates can be scheduled pretty far in advance, so I'd definitely add three years into the future. 

It doesn't sound like PG CASA has any plans (right now, anyway) to backlog, but I think there will be enough of a use case potentially beyond PG CASA that we should make the lower year into the early 2000s.

Reply:
Thanks for the feedback @littleforest and @Zrrrpy. I've made it from 3 years in the future to 2000... sorting it backwards keeps the current year visible in the drop down.
![Screenshot from 2020-10-04 10-55-07](https://user-images.githubusercontent.com/1674861/95003927-70775e00-0630-11eb-941c-98df30184bff.png)